### PR TITLE
Fix/email template

### PR DESCRIPTION
**Pull request by alessio-creo**

### CODE REVIEW

File: .vscode/settings.json
Comment (micdes-pagopa) [inline .vscode/settings.json:1]: Remove this file

File: .metals/metals.mv.db
Comment (micdes-pagopa) [inline .metals/metals.mv.db:1]: Remove this file

File: .metals/metals.log
Comment (micdes-pagopa) [inline .metals/metals.log:1]: Remove this file

File: .metals/metals.lock.db
Comment (micdes-pagopa) [inline .metals/metals.lock.db:1]: Remove this file

Comment (alessio-creo) [reply .metals/metals.lock.db:1]: Done all the above

**Pull request merged**